src/Entity/Adherent.php

```diff
@@ -452,20 +452,45 @@ public function getEmailsSubscriptions(): array
         if ($this->hasCitizenProjectCreationEmailSubscription()) {
             $subscriptions[] = AdherentEmailSubscription::SUBSCRIBED_EMAILS_CITIZEN_PROJECT_CREATION;
         }
+//      CANARY !
+        if ($this->localHostEmailsSubscription) {
+            $subscriptions[] = AdherentEmailSubscription::SUBSCRIBED_EMAILS_LOCAL_HOST;
```

Comment:
`LOCAL_HOST` => `LOCALHOST` :trollface:

Reply:
:D I know ^^, it's one of the old fields removed ^^,